Update adguard from 1.5.14 to 2.0.4

SharjeelJ opened the pull request:
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

vitorgalvao commented:
> It seems the App source '/usr/local/Caskroom/adguard/2.0.4/Adguard.app' is not there.

How to fix: [Reporting bugs](https://github.com/Homebrew/homebrew-cask#reporting-bugs) → [`source is not there` error](https://github.com/Homebrew/homebrew-cask/blob/master/doc/reporting_bugs/source_is_not_there_error.md).

SharjeelJ commented:
I Believe I fixed it. The original was just a .app in the root of the .dmg whereas the new format is a .pkg in the root of the .dmg (adjusted from app -> pkg). I also added in the "uninstall pkgutil" line with the correct pkg identifier. All in all, let me know if anything I did is done wrong / could be improved so I can do so in the future :)

vitorgalvao commented on Casks/adguard.rb:
```diff
@@ -1,13 +1,17 @@
-  app 'Adguard.app'
+  pkg 'AdGuard.pkg'
+
+  uninstall pkgutil: [
```
Since it’s only one element, it can be a string. I.e. `uninstall pkgutil: 'com.adguard.mac.adguard-pkg'`.

SharjeelJ replied:
Adjusted it, thank you for the tip :)

vitorgalvao commented:
Great job! Thank you. We’ll be able to merge this as soon as Travis CI passes (again).
